packages/lib-classifier/src/plugins/tasks/survey/components/components/Choice/Choice.js

```diff
@@ -85,57 +114,92 @@ function Choice({
-          setAnswers={handleAnswers}
-          strings={strings}
-        />
-      )}
       <Box
-        border={{
-          color: 'light-5',
-          side: 'top',
-          size: 'small'
+        pad={{
+          horizontal: '20px',
+          vertical: '30px'
         }}
-        direction='row'
-        fill='horizontal'
-        gap='xsmall'
-        justify='center'
-        margin={{ top: 'small' }}
-        pad={{ top: 'small' }}
       >
-        <Button
+        <Box
+          direction='row'
           fill='horizontal'
-          label={t('SurveyTask.Choice.notThis')}
-          onClick={() => handleDelete(choiceId)}
-        />
-        <PrimaryButton
-          data-testid='choice-identify-button'
-          disabled={!allowIdentify}
+          justify='between'
+          margin={{ bottom: '30px' }}
+        >
+          <Heading
+            id='choice-label'
+            color={{
+              dark: 'accent-1',
+              light: 'neutral-7'
+            }}
+            margin='none'
+            size='1.5rem'
+            weight='bold'
+          >
+            {strings.get(`choices.${choiceId}.label`)}
+          </Heading>
+          <Button
+            a11yTitle={showInfo ? t('SurveyTask.Choice.lessInfo') : t('SurveyTask.Choice.moreInfo')}
+            label={InfoLabel}
+            onClick={() => setShowInfo(!showInfo)}          
+            plain
+          />
+        </Box>
+        <Collapsible
+          open={showInfo}
+        >
+          <Box margin={{ bottom: '30px' }}>
+            <Paragraph margin='none'>
+              {strings.get(`choices.${choiceId}.description`)}
+            </Paragraph>
+            {choice.confusionsOrder?.length > 0 && (
+              <ConfusedWith
+                choices={choices}
+                choiceId={choiceId}
+                confusions={choice.confusions}
+                confusionsOrder={choice.confusionsOrder}
+                handleChoice={handleChoice}
+                images={images}
+                strings={strings}
+              />
+            )}
+          </Box>
+        </Collapsible>
+        {questionIds.length > 0 && (
+          <Questions
+            answers={answers}
+            questionIds={questionIds}
+            questions={questions}
+            setAnswers={handleAnswers}
+            strings={strings}
+          />
+        )}
+        <Box
+          border={{
+            color: 'light-3',
+            side: 'top',
+            size: 'small'
+          }}
+          direction='row'
           fill='horizontal'
-          label={t('SurveyTask.Choice.identify')}
-          onClick={() => onIdentify()}
-        />
+          gap='xsmall'
+          justify='center'
+          margin={{ top: '30px' }}
+          pad={{ top: 'small' }}
+        >
+          <StyledButton
+            fill='horizontal'
+            label={t('SurveyTask.Choice.cancel')}
+            onClick={() => handleDelete(choiceId)}
+          />
```

Comment:
Previously, if a previously selected choice was selected again, then "Not this" deleted that choice selection. Should "Cancel" still have the same delete choice selection action for previously selected choices?

Reply:
Resolved in [090eeb9](https://github.com/zooniverse/front-end-monorepo/pull/6478/commits/090eeb95d121c4ffdff3f938c483760e69834e0b).